docs/add_kafka_topics.md

```diff
@@ -0,0 +1,32 @@
+# Add Kafka Topics
+
+To add Kafka topics, find your target environment under `overlays` and create a new `KafkaTopic` resource within the
+`kafka` sub directory. For example if you would like to add your `KafkaTopic` on `MOC` then first create a topic like
+the one below:
+
+Replace `my-topic` with a preferred name.
+```yaml
+# my-topic.yaml
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: orders
+  labels:
+    strimzi.io/cluster: odh-message-bus
+spec:
+  partitions: 2
+  replicas: 3
+```
+
+The label `strimzi.io/cluster` should have the value `odh-message-bus`. The recommended partition size is `2` and
+replica count is `3`. Pick a suitable name, ensure that it's unique in the `topics` folder.
```

Comment:
You might also want to mention here that the replica count should be lower than/equal to the number of Kafka brokers

Reply:
updated, thanks!